wellcomeml/__main__.py

```diff
@@ -0,0 +1,26 @@
+import subprocess
+import sys
+import os
+
+
+VENV_PYTHON = os.path.join(
```

Comment:
I'd still like `VENV_PYTHON` to be able to be configured by the user (and possibly defaulted to plain `python`). The problem with the current approach is that users will have to have a virtualenv named exactly `build/virtualenv/bin/python`. For example. if I install wellcomeml in a virtualenv called `venv`, this command will not allow me to download models...

Reply:
hmm, that is a very good point. in this case i think we should go back to python and delegate to the user i.e. if the user is in a virtualenv then it will install there otherwise it will install wherever python is. that makes more sense. the make command should assume venv is created with make so that is fine as well. the only problem to resolve are the tests in travis.

Reply:
one way to fix that is to change the make command actually to not use the download utility and install models which would be duplicate and we would need to maintain which models are downloaded in two places. alternatively i am wondering whether we can prepend the activation of the environment as a hack?

Reply:
That's true. I think it is ok to assume the user will be in the correct environment if he runs the command ad-hoc. As for the makefile, we can replace 

https://github.com/wellcometrust/WellcomeML/blob/b52fb5cd1864ff04eff4488ae432727cd84c0996/Makefile#L44

to
```bash
. $(VIRTUALENV)/bin/python && $(VIRTUALENV)/bin/python -m spacy download en_core_web_sm
```
I believe it should work, but it has to be done in the same line, because otherwise `make` will run it in a different subprocess.